website/sidebars.json

```diff
@@ -1,6 +1,6 @@
 {
     "docs": {
-        "Introduction": ["accordproject", "accordproject-concepts", "accordproject-business", "accordproject-developers"],
+        "Introduction": ["accordproject", "accordproject-concepts", "accordproject-business", "faq-lawyers","accordproject-developers"],
```

Comment:
I'm a little bit confused about where the FAQ should go.

Having a new Section `FAQ` makes sense to me? If that's the preferred option, then probably let's not have a different FAQ entry in the Introduction (so remove this one, but keep the `FAQ` below in the same file).

Reply:
It is a little redundant to have the overview and the faq stuff I presented and I like the overview -- I also like addressing lawyers directly with their concerns  - It might make more sense to go back and look at the Overview and maybe sprinkle little faqs through the documentation. It seems to me that the better the docs the less you need a faq. I'm willing to move to something else